config/routes.rb

```diff
@@ -11,6 +11,7 @@
   post "/feedback" => "feedback#create"
 
   match "/auth/:provider/callback" => "auth#oauth_callback", via: [:get]
+  get "/auth/failure" => "auth#failure"
```

Comment:
yeah I am think it is OK for us not to know what causes it. Just thinking it would be nice to have some sort of test failure in the case that someone either deleted or updated this route incorrectly.